member_map_internal_test.go

```diff
@@ -22,6 +22,130 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+func TestMemberMap_Alive_Internal(t *testing.T) {
+	m := NewMemberMap()
+
+	//when member 1 suspectedMember
+	//when Incarnation Number of Parameter of Alive Func is bigger than member in memberList
+	m.members[MemberID{ID: "1"}] = Member{
+		ID: MemberID{
+			ID: "1",
+		},
+		Incarnation: 1,
+		Status:      Suspected,
+	}
+	isChanged, err := m.Alive(AliveMessage{
+		MemberMessage: MemberMessage{
+			ID:          "1",
+			Incarnation: 1,
+		},
+	})
+
+	//Then
+	assert.Nil(t, err)
+	assert.Equal(t, true, isChanged)
+	assert.Equal(t, Alive, m.members[MemberID{ID: "1"}].Status)
+
```

Comment:
`assert.Equal(t, Alive, m.members[MemberID{ID: "1"}].Status)` this should be
`assert.Equal(t, Suspected, m.members[MemberID{ID: "1"}].Status)`